src/components/footer/index.tsx

```diff
@@ -0,0 +1,90 @@
+"use client";
+
+import React, { useState } from "react";
+import { Box, Section, Text } from "@radix-ui/themes";
+import Image from "next/image";
+import CollapsibleBox from "../boxes/collapsible-box/collapsible-box";
+import {
+  BOX1_ITEMS,
+  BOX2_ITEMS,
+  BOX3_ITEMS,
+} from "../../constants/footer-items-mock";
+import footerLogo from "../../../public/footer-logo.png";
+import Link from "next/link";
+import facebook from "../../../public/facebook.png";
+import teleg from "../../../public/telegram.png";
+import insta from "../../../public/insta.png";
+
+const Footer: React.FC = () => {
+  const [visibleBox, setVisibleBox] = useState<string | null>(null);
+
+  const handleToggle = (box: string) => () => {
+    setVisibleBox((prev) => (prev === box ? null : box));
+  };
+
+  return (
+    <Box className="bg-footer-bg text-white py-8">
+      <Section className="container mx-auto px-4 py-0 flex flex-col md:flex-row md:justify-between">
+        <Box className="flex-1 mb-4 text-white flex flex-col items-center ">
+          <Image
+            src={footerLogo}
+            alt="Logo"
+            className="mb-2"
+            width={130}
+            height={70}
+          />
+          <Text
+            className="text-lg text-center  mt-4"
+            style={{ color: "white" }}
+          >
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
```

Comment:
Please all literals put to the separate fiel

Reply:
done